Platform/TigerlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -2415,9 +2419,27 @@ UpdateCpuNvs (
     CpuNvs->DtsIoTrapLength = DTS_IO_TRAP_LENGTH;
     CpuNvs->DtsAcpiEnable = DTS_ACPI_DISABLE;
   }
+
+  ///
+  /// Update TXT status for ACPI
+  ///
+  CpuNvs->TxtEnabled = 0;  // Default to disabled
+  if (FeaturePcdGet (PcdTxtEnabled)) {
+    FEATURES_CFG_DATA *FeaturesCfgData;
+    FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag(CDATA_FEATURES_TAG);
+    if ((FeaturesCfgData != NULL) && (FeaturesCfgData->Features.TxtEnabled == 1)) {
+      if (IsTxtEnabled() == TRUE) {
```

Comment:
IsTxtEnabled() returns a BOOLEAN value, it is not necessary compare with another BOOLEAN.